src/Microsoft.HttpRepl.Fakes/MockHttpMessageHandler.cs

```diff
@@ -9,16 +11,46 @@ namespace Microsoft.HttpRepl.Fakes
 {
     public class MockHttpMessageHandler : HttpMessageHandler
     {
-        private HttpResponseMessage response;
+        private string _fileContents;
+        private string _filePath;
```

Comment:
`_filePath` is never used after it is set. Is that a mistake, or can it be removed?

Reply:
Yes, it not used. Removed it here:
https://github.com/aspnet/HttpRepl/commit/e438bd54448a497c75a978e6bd7cc95441349948

Thanks!